library/aws/rds.ts

```diff
@@ -99,7 +99,7 @@ export function createPostgresInstance(
     publicly_accessible: false,
     backup_retention_period: 3,
     vpc_security_group_ids: [security_group.id],
-    db_subnet_group_name: db_subnet_group.name,
+    db_subnet_group_name: db_subnet_group ? db_subnet_group.name: undefined,
```

Comment:
You can use shortcut `db_subnet_group?.name`

Reply:
Not sure how that works but... changed